package.json

```diff
@@ -19,62 +19,60 @@
   "description": "Connect allthethings",
   "devDependencies": {
     "archiver": "^1.3.0",
-    "autoprefixer": "^6.7.4",
-    "babel-eslint": "^7.1.1",
-    "babel-plugin-add-module-exports": "^0.2.1",
-    "babel-plugin-istanbul": "^4.0.0",
-    "babel-plugin-lodash": "^3.2.11",
-    "babel-plugin-transform-imports": "^1.4.1",
-    "babel-polyfill": "^6.23.0",
     "colors": "^1.1.2",
     "confirm-simple": "^1.0.3",
-    "cross-env": "^3.1.4",
-    "css-loader": "^0.26.1",
-    "eslint": "3.16.0",
-    "eslint-config-prettier": "^2.9.0",
-    "eslint-config-wpcalypso": "^2.0.0",
-    "eslint-loader": "^1.6.1",
-    "eslint-plugin-import": "^2.9.0",
-    "eslint-plugin-jest": "^21.2.0",
-    "eslint-plugin-jsx-a11y": "^6.0.3",
-    "eslint-plugin-react": "^7.1.0",
-    "eslint-plugin-wpcalypso": "^3.4.1",
-    "exports-loader": "^0.6.3",
-    "extract-text-webpack-plugin": "^2.0.0-rc.3",
-    "file-loader": "^0.10.0",
     "globby": "^6.1.0",
-    "imports-loader": "^0.7.0",
-    "json-loader": "^0.5.4",
-    "node-sass": "4.9.0",
     "po2json": "^0.4.5",
-    "postcss-loader": "^1.3.1",
-    "sass-loader": "^6.0.2",
     "shelljs": "^0.7.6",
-    "style-loader": "^0.13.1",
-    "svg-url-loader": "^2.0.2",
-    "url-loader": "^0.5.7",
-    "webpack": "^2.2.1",
-    "webpack-dev-server": "^2.4.1",
-    "webpack-node-externals": "^1.5.4",
-    "webpack-notifier": "^1.5.0",
     "xgettext-js": "^1.1.0"
   },
   "dependencies": {
```

Comment:
I'm putting all the dependencies I touched in `dependencies` because before we weren't being consistent in our use of depencencies vs devDependencies. I think it's better to not separate them at all, than to separate them arbitrarily 🤷‍♂️

Reply:
The line is not as clear-cut as you think it is :)

I'd say that testing and linting are `devDependencies`, but since we (currently) use the ESLint Webpack loader, so `npm start` or even `npm run dist` would fail without `eslint`.

I would consider transpiling (babel) a `dependency`, because it's needed in order to produce a `production` bundle.

Anyway, I'm trying to remove almost all our dependencies entirely and ge tthe build to use the ones under `node_modules/wp-calypso/node_modules` instead, so in the near future we may end up with just a handful of dependencies, and we can decide then.